keyboards/annepro2/keymaps/default-full-caps/keymap.c

```diff
@@ -0,0 +1,127 @@
+#include <stdint.h>
+#include "annepro2.h"
+#include "qmk_ap2_led.h"
+#include "config.h"
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

And:
```suggestion
#include QMK_KEYBOARD_H
```